lib/actions/user.js

```diff
@@ -203,7 +192,7 @@ export function deleteUserMonitoredTrip (id) {
 
       if (deleteResult.status === 'success') {
         // Reload user's monitored trips after deletion.
-        await dispatch(fetchUserMonitoredTrips(accessToken))
+        dispatch(fetchUserMonitoredTrips(accessToken))
```

Comment:
accessToken arg needs to be removed.

Reply:
Removed in https://github.com/opentripplanner/otp-react-redux/pull/233/commits/9b74a576f40dd13476fa8c0f4a218dfe55dd6dd5